components/browser/state/src/main/java/mozilla/components/browser/state/state/MediaSessionState.kt

```diff
@@ -0,0 +1,30 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.browser.state.state
+
+import mozilla.components.concept.engine.mediasession.MediaSession
+
+/**
+ * Value type representing a media session on a website.
+ *
+ * @property controller The [MediaSession.Controller] for controlling playback of this media session.
+ * @property metadata The [MediaSession.Metadata] for this media session.
+ * @property elementMetadata The [MediaSession.ElementMetadata] for this media session.
+ * @property playbackState The current simplified [MediaSession.PlaybackState] of this media session.
+ * @property features The [MediaSession.Feature] for this media session.
+ * @property positionState The current simplified [MediaSession.PositionState] of this media session.
+ * @property muted True if media session is muted.
+ * @property fullscreen True if media session is fullscreen.
+ */
+data class MediaSessionState(
+    val controller: MediaSession.Controller? = null,
```

Comment:
It looks like we will get the controller with the activate callback, so this will probably be never null?

Reply:
Yes, I've confirmed that there's no usecase that will create a `MediaSessionState` without controller.  I'll fix.  Thanks,